src/flows/OnboardingFlow/steps/AddUserInfo.tsx

```diff
@@ -19,8 +20,6 @@ function useWizardConfig({ onNext }: ConfigProps) {
 
   useEffect(() => {
     setOnNext(onNext);
-
-    return () => setOnNext(undefined);
```

Comment:
I removed this because it was resetting the next step's next(), since it was being called after the next step had already called `setOnNext()`.